.erb/configs/webpack.config.renderer.dev.ts

```diff
@@ -149,6 +150,7 @@ const configuration: webpack.Configuration = {
     __filename: false,
   },
 
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
   // @ts-ignore
```

Comment:
This is quite ugly, but it's not obvious to me how to get rid of this `@ts-ignore`.